### PR TITLE
some docker fixups that @aNullValue discovered

### PR DESCRIPTION
**Pull request by jafow**

### CODE REVIEW

**Review by jmensch1 — COMMENTED** [review]
wondering why we're taking out the entrypoint script? I though we were gonna us that to trigger the migrations.

**Comment by jafow** [comment]
> wondering why we're taking out the entrypoint script? I though we were gonna us that to trigger the migrations.

I saw that you added a db.sync promise to the server run and thought that handles migration checks. I think that's actually a nice solution to put it in the server. But maybe doesn't actually work that way?
If not then yes I'll add back in as we do want them run on boot up

**Comment by jmensch1** [comment]
Ok, yeah I was thinking that the sync would be temporary. Once we get some real data I think we're gonna need to switch to the migrations.

**Comment by jafow** [comment]
> Ok, yeah I was thinking that the sync would be temporary. Once we get some real data I think we're gonna need to switch to the migrations.

cool cool :+1:  i added back in

File: backend/docker-compose.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ services:
     command: npm run dev
 
   db:
-    image: postgres:11
+    image: jred/postgres:11-postgis
```
Comment (jafow) [inline backend/docker-compose.yml:21]: Ideally we move this to a hackforla or better a 100Automations dockerhub repo but i just pushed to mine own for now.

Comment (jafow) [reply backend/docker-compose.yml:21]: the source for it is herehttps://github.com/100Automations/dockerfiles/tree/master/postgresql/11-postgis

File: backend/entrypoint.sh [hunk 2]
```diff
@@ -3,11 +3,25 @@ set -e
 
 # check if we should run migrations
 if [ -z $DISABLE_DB_MIGRATION_AUTO_RUN ]; then
+    printf "%s\n" "======================================"
```
Comment (jafow) [inline backend/entrypoint.sh:6]: i was gonna try to make american flag with stars and stripes but my ascii art is -1 and probably better things to focus on now :us:

File: backend/docker-compose.yml (hunk 1 above)
Comment (jmensch1) [reply backend/docker-compose.yml:21]: nice!

**Review by jmensch1 — APPROVED** [review]

**Pull request merged**